web/src/lib/components/shared-components/scrollbar/scrollbar.svelte

```diff
@@ -23,25 +24,35 @@
   $: scrollY = toScrollY(timelineY);
 
   class Segment {
-    public count;
-    public height;
-    public timeGroup;
-
-    constructor({ count = 0, height = 0, timeGroup = '' }) {
-      this.count = count;
-      this.height = height;
-      this.timeGroup = timeGroup;
-    }
+    public count: number = 0;
+    public height: number = 0;
+    public timeGroup: string = '';
+    public date!: DateTime;
```

Comment:
I don't think `!:` is necessary here, is it?

Reply:
Yes, it is.